base_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable
```

Comment:
- [ ] The correct_name method  should return the result of the correct_name method of  @namable, like so:
 ```ruby
    def correct_name
      @nameable.correct_name
    end
```
Please kindly fix that.

Reply:
thanks for pointing that out.